ci/install.sh

```diff
@@ -24,3 +24,23 @@ gem install --no-document --version ">= 0.8.0" neovim
 echo "Install neovim npm package"
 npm install -g neovim
 npm link neovim
+
+echo "Install tree-sitter npm package"
+npm install -g tree-sitter-cli
+npm link tree-sitter-cli
+
+echo "Install tree-sitter c parser"
+curl "https://codeload.github.com/tree-sitter/tree-sitter-c/tar.gz/v0.15.0" -o tree_sitter_c.tar.gz
+tar xf tree_sitter_c.tar.gz
+cd tree-sitter-c-0.15.0
+export TREE_SITTER_DIR=$HOME/tree-sitter-build/
+mkdir -p $TREE_SITTER_DIR/bin
+
+if [[ "$BUILD_32BIT" != "ON" ]]; then
+  # builds c parser in $HOME/tree-sitter-build/bin/c.(so|dylib)
+  tree-sitter test
+else
+  # no tree-sitter binary for 32bit linux, so fake it (no tree-sitter unit tests)
+  cd src/
+  gcc -m32 -o $TREE_SITTER_DIR/bin/c.so -shared parser.c -I.
+fi
```

Comment:
would be nice to put this treesitter setup in a separate function since it is quite involved. (Same for `ci/build.ps1`)

Reply:
The rest of the code does not use functions (including python setup and other specific sections which are substantially longer). So I do not know what style or naming convention such a function should be, or what the arguments would be etc. I just wrote code that did the necessary setup here, restructuring the ci setup code seems like a separate PR to me.

Reply:
[common/build.sh](https://github.com/neovim/neovim/blob/a7a747ef6948e902929b7bbde4d2140654e73824/ci/common/build.sh) might be a good place.